### ISSUE TITLE
Hide observed cell counts in contingency tables when checkbox is cleared

### ISSUE THREAD

**Issue by HorwoodP** [jasp-stats/jasp-issues#80]
* JASP version: 0.8.2
* OS name and version:  Windows 10, version 1607

I've noticed that Observed cell counts are displayed in contingency table regardless of whether the Observed checkbox is ticked or not.

Would it be possible to suppress the display of Observed cell counts when another of the cell display options are ticked (e.g. Column Percentages) and the Observed box is not ticked?

**Comment by tomtomme** [comment]
@HorwoodP 
The checkbox for observed counts was removed as an option. I do not know why. 

@ASarafoglou 
The request was, to have a working checkbox for observed counts. Was there any trouble implementing this? Honest question. I do know almost nothing about coding.

**Comment by tomtomme** [comment]
@HorwoodP @FBartos 
Do you still consider this an issue now that the checkbox is removed?
Is the removal of observed counts really needed as an option? I do not see a usecase yet.

**Comment by HorwoodP** [comment]
Thanks for following this issue up.
Being able to remove observed counts and just display percentages is a requirement of survey reporting, so I still need to use something like SPSS Custom Tables rather than JASP when generating tables of survey results (unless I want to manually edit the JASP output to remove the observed counts).

**Comment by FBartos** [comment]
Hi @HorwoodP , could you please maybe share what the desired table is supposed to look like? (I'm not completely sure whether you still wanna column/row sums etc...)

**Comment by FBartos** [comment]
I can add something like this now:
both Observed & Row-wise Percentages:
![image](https://github.com/user-attachments/assets/44ccf705-1dad-465c-a0f2-e68bea9e02f8)
Only Row-wise percentages
![image](https://github.com/user-attachments/assets/cdee011b-67a3-4394-853f-8fcefeaacd78)

**Comment by tomtomme** [comment]
Looks good!

**Comment by HorwoodP** [comment]
Hi,

My use case (6 years ago) would be just showing the % of column values for
survey reporting. Back then I was using SPSS Custom Tables to produce a
simple table output like the attached image.



Phil Horwood
mob:  0402 483 710

**Comment by FBartos** [comment]
Yeah, sorry for taking this long on the issue. The image didn't get through but also added toggling the margins on/off.

**Comment by HorwoodP** [comment]
Awesome, thanks for the update!

Phil Horwood
mob:  0402 483 710
